swim/node.go

```diff
@@ -58,7 +58,9 @@ type Options struct {
 func defaultOptions() *Options {
 	opts := &Options{
 		StateTimeouts: StateTimeouts{
-			Suspect: 5000 * time.Millisecond,
+			Suspect:   5 * time.Second,
+			Faulty:    1 * time.Hour,
```

Comment:
How did you arrive at 1 hour as a default for tombstone? I had it in my mind that it would be more like 24 hours...